node_type: eagerx_core.node/RealResetNode
default:
  rate:
  inputs:
  outputs:
  states:
  feedthroughs:
  single_process: False
  launch_locally: True
  launch_file: '$(find eagerx_core)/launch/rxnode.launch'
  test_arg: 'TEST ARGUMENT'

inputs:
  in_1:
    msg_type: std_msgs.msg/UInt64
    repeat: all
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]
  in_2:
    msg_type: std_msgs.msg/UInt64
    repeat: all
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]

outputs:
  out_1:
    msg_type: std_msgs.msg/UInt64
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]
  out_2:
    msg_type: std_msgs.msg/UInt64
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]

states:
  state_1:
    msg_type: std_msgs.msg/UInt64

feedthroughs:
  out_1:
    msg_type: std_msgs.msg/UInt64
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]
  out_2:
    msg_type: std_msgs.msg/UInt64
    gym_space:
      converter_type: eagerx_core.converter/SpaceUInt64Converter
      dtype: boxu64
      low: [0]
      high: [100]


